libs/payments/cart/src/lib/cart.types.ts

```diff
@@ -1,41 +1,45 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-import { Cart as CartDB, CartFields } from '@fxa/shared/db/mysql/account';
+import { Cart } from '@fxa/shared/db/mysql/account';
 
+export type ResultCart = Readonly<Omit<Cart, 'id' | 'uid'>> & {
+  readonly id: string;
+  readonly uid?: string;
+};
 export interface TaxAmount {
   title: string;
   amount: number;
 }
 
-export interface Invoice {
-  totalAmount: number;
-  taxAmounts: TaxAmount[];
-}
-
-export type Cart = CartFields & {
-  previousInvoice?: Invoice;
-  nextInvoice: Invoice;
+export type SetupCart = {
+  uid?: string;
+  interval: string;
+  offeringConfigId: string;
+  experiment?: string;
+  taxAddress?: Record<string, string>;
```

Comment:
Is it possible to be more specific in the typing of taxAddress?

Reply:
Yep, this can be a more specific Json type, which kysley made. Will follow up with another polish PR for that.